src/rules/comment-word-blacklist/__tests__/index.js

```diff
@@ -6,6 +6,30 @@ const rule = rules[ruleName]
 
 testRule(rule, {
   ruleName,
+  config: ["bad-word"],
+
+  accept: [ {
+    code: "/* comment */",
+  }, {
+    code: "/*! bad-word */",
+  }, {
+    code: "/*# bad-word */",
+  }, {
+    code: "/** bad-word **/",
+  }, {
+    code: "/*** bad-word ***/",
+  } ],
```

Comment:
Maybe I missed this earlier ... but these tests seem problematic to me. Why aren't they complaining about `bad-word`? Those are valid comments, right?